k8s/base/daemon-config.json

```diff
@@ -0,0 +1,34 @@
+{
+    "anchor": {},
+    "http-api": {
+        "cors-allowed-origins": [
+            ".*"
+        ],
+        "admin-dids": [
+            "${CERAMIC_ADMIN_DID}"
+        ]
+    },
+    "ipfs": {
+        "mode": "remote",
+        "host": "${CERAMIC_IPFS_HOST}"
+    },
+    "logger": {
+        "log-level": 2,
```

Comment:
Configmap?

Reply:
A configmap is generated from the file with the configmapgenerator.

Reply:
I'm suggesting that this and the cors setting should not be hardcoded, but should instead come from a different source.

Reply:
The settings are in a file that gets redeployed when changed and restarts the workload. How is that "hard coded"? 
If you want to change it, change the file and redeploy.

What other source would the configurations come from? Further abstraction doesn't add value if the settings aren't used elsewhere.

Reply:
I'm looking at how we would deploy this. Ideally, we wouldn't change this file and redeploy, but rather change the a file that just deals with configurable values. We could just hardcode CERAMIC_IPFS_HOST and change this for every deploy, but instead we have values that go along with the deploy. For logging and cors, we would have something similar, where they should live outside of this file, so they can be set on a per environment, or at deploy time.